src/git.rs

```diff
@@ -51,14 +51,32 @@ impl GitHelper {
     /// Arguments:
     ///
     /// - rev: A single commit rev spec
-    /// - prefix: The version prefix
-    pub(crate) fn find_last_version(&self, rev: &str) -> Result<Option<VersionAndTag>, Error> {
+    /// - ignore_prereleases: If true, ignore pre-release versions
+    pub(crate) fn find_last_version(
+        &self,
+        rev: &str,
+        ignore_prereleases: bool,
+    ) -> Result<Option<VersionAndTag>, Error> {
         let rev = self.repo.revparse_single(rev)?.peel_to_commit()?;
         let mut revwalk = self.repo.revwalk()?;
         revwalk.push(rev.id())?;
         let mut version: Vec<&VersionAndTag> = revwalk
             .flatten()
-            .filter_map(|oid| self.version_map.get(&oid))
+            .filter_map(|oid| {
+                let version = self.version_map.get(&oid);
+
+                if ignore_prereleases {
+                    if let Some(version) = version {
+                        if !version.version.is_prerelease() {
+                            return Some(version);
+                        }
+
+                        return None;
+                    }
+                }
+
+                version
```

Comment:
This can probably be simplified:

```suggestion
                self.version_map
                    .get(&oid)
                    .filter(|v| !ignore_prereleases || !v.version.is_prerelease())
```

Reply:
Thanks, looking better!